platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/BaseBench.java

```diff
@@ -246,11 +244,15 @@ protected long nextValue() {
         return Utils.randomLong();
     }
 
-    public BenchmarkConfig getConfig() {
-        return getConfig(BenchmarkConfig.class);
+    public Configuration getConfig() {
```

Comment:
This method looks redundant.

Reply:
Addressed, see [this commit](https://github.com/hashgraph/hedera-services/pull/15562/commits/b321af77b1425e7c5b0575d6216817ec1f32c005)